Clients/GUI/ViewModels/SignUpViewModel.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.ComponentModel.DataAnnotations;
 
-namespace GUI.ViewModels
+namespace GUI.GUIModels
```

Comment:
I suppose we need to rename the method(s) as well

Reply:
what name is your favorit?

Reply:
Not favourite, but uniform. We've renamed the Namespace to _GUIModels_, so why methods are still _...ViewModel..._?

Reply:
I have no idea why we renamed it to GUIModels, it should be ViewModels

Reply:
Fix it please